python/pandemic_simulator/environment/regulation_stages.py

```diff
@@ -102,86 +115,106 @@
+                        Restaurant: {
+                            'open_time': SimTimeTuple(hours=(), week_days=())},
+                        Bar: {
+                            'open_time': SimTimeTuple(hours=(), week_days=())},
+                    },
+                    stage=3),
+    CovidRegulation(stay_home_if_sick=True,
+                    practice_good_hygiene=True,
+                    wear_facial_coverings=True,
+                    social_distancing=0.8,
+                    risk_to_avoid_gathering_size={Risk.HIGH: 0, Risk.LOW: 0},
+                    location_type_to_rule_kwargs={
+                        Office: {'lock': True},
+                        School: {'lock': True},
+                        BarberShop: {'lock': True},
+                        RetailStore: {'lock': True},
+                        Restaurant: {'open_time': SimTimeTuple(hours=(), week_days=())},
+                        Bar: {
+                            'open_time': SimTimeTuple(hours=(), week_days=())},
+                    },
+                    stage=4)
 ]
 
-# https://home.kpmg/xx/en/home/insights/2020/04/sweden-government-and-institution-measures-in-response-to-pandemic.html
+# https://home.kpmg/xx/en/home/insights/2020/04/sweden-government-and-institution-measures-in-response-to-covid.html
 # Sweden took no nationwide lockdown; Remote work *recommended*;
 # Schools are open; Restaurants are open.
 # Travel ban.
 
-# https://www.folkhalsomyndigheten.se/the-public-health-agency-of-sweden/communicable-disease-control/pandemic-19/prevention/
+# https://www.folkhalsomyndigheten.se/the-public-health-agency-of-sweden/communicable-disease-control/covid-19/prevention/
 # We do not currently recommend face masks in public settings since the scientific evidence
 # around the effectiveness of face masks in combatting the spread of infection is unclear.
 # https://www.folkhalsomyndigheten.se/the-public-health-agency-of-sweden/communicable-disease-control/
-# pandemic-19--the-swedish-strategy/
+# covid-19--the-swedish-strategy/
 
 # Anders Tegnell says his modelling indicates that, on average, Swedes have around 30% of the social interactions they
 # did prior to the pandemic.
-swedish_regulations: List[PandemicRegulation] = [
-    PandemicRegulation(stage=0),
-    PandemicRegulation(stay_home_if_sick=True,
-                       practice_good_hygiene=True,
-                       wear_facial_coverings=False,
-                       social_distancing=0.70,
-                       risk_to_avoid_gathering_size={Risk.HIGH: 50, Risk.LOW: 50},
-                       location_type_to_rule_kwargs={
-                           Office: {'lock': False},
-                           School: {'lock': False},
-                           BarberShop: {'lock': False},
-                           RetailStore: {'lock': False},
-                       },
-                       stage=1),
+swedish_regulations: List[CovidRegulation] = [
```

Comment:
Not sure why you changed this back to `CovidRegulation`. It should be `PandemicRegulation` (@rcapobianco right?)